src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnectionBuilderExtensions.cs

```diff
@@ -23,7 +23,8 @@ public static IHubConnectionBuilder WithJsonProtocol(this IHubConnectionBuilder
 
         public static IHubConnectionBuilder WithMessagePackProtocol(this IHubConnectionBuilder hubConnectionBuilder)
```

Comment:
Add overloads of this and JsonProtocol that take the settings so this can be configured on the client as well?

Reply:
OK. Was on the fence if want it or not.